src/openklant/utils/serializers.py

```diff
@@ -0,0 +1,46 @@
+from django.utils.module_loading import import_string
+
+from rest_framework_nested.serializers import NestedHyperlinkedRelatedField
+
+
+class ExpandSerializer(NestedHyperlinkedRelatedField):
```

Comment:
I think this should be moved to vng-api-common once https://github.com/open-zaak/open-zaak/pull/1037 is reviewed/merged

Reply:
Lijkt me wel, ik zou bij beide een # TODO hiervoor toevoegen